boards/arm/usb_kw24d512/doc/usb_kw24d512.rst

```diff
@@ -0,0 +1,127 @@
+.. _usb_kw24d512:
+
+NXP USB-KW24D512
+################
+
+Overview
+********
+
+The USB-KW24D512 board is an IEEE 802.15.4 packet sniffer in a
+convenient USB dongle form factor.
+
+- Packet sniffers are essential for wireless network development allowing
```

Comment:
This would read better just as regular sentences (with the previous one) rather than two bullet points.